src/serializer/MapFishPrintV3OSMSerializer.ts

```diff
@@ -1,51 +1,28 @@
-import OlSource from 'ol/source/Source';
-import OlSourceOSM from 'ol/source/OSM';
 import OlLayer from 'ol/layer/Layer';
+import OlSource from 'ol/source/Source';
+import olSourceOSM from 'ol/source/OSM';
```

Comment:
```suggestion
import OlSourceOSM from 'ol/source/OSM';
```

Reply:
it's still `olSourceOSM` - maybe you didn't apply the suggestion?

Reply:
done in: [5fb9911](https://github.com/terrestris/mapfish-print-manager/pull/482/commits/5fb9911fc3f7f58244edf255f35fb900da41d7bf)